pkg/cmd/server/api/v1/types.go

```diff
@@ -626,3 +637,16 @@ type PodManifestConfig struct {
 	// The interval needs to be a positive value
 	FileCheckIntervalSeconds int64 `json:"fileCheckIntervalSeconds"`
 }
+
+type ExtensionFilesConfig struct {
+	// SubContext is the path under /<context>/extensions/ to serve files from Location
+	SubContext string `json:"subcontext"`
+	// Location is a directory on the asset server to serve files under SubContext in the
+	// Web Console. It may have nested folders.
+	Location string `json:"location"`
```

Comment:
Yes, I tried to name in something generic so it could be a URL later on. I'm good with `sourceDirectory`.